region/norfair/east.json

```diff
@@ -3372,7 +3414,18 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                    {"heatFrames": 250}
+                    {"or": [
+                      {"heatFrames": 230},
+                      {"and": [
+                        "canDamageBoost",
+                        {"heatFrames": 200},
+                        {"enemyDamage": {
```

Comment:
fwiw I make it out with more health with a contact damage boost by dropping faster.  By waiting for the particles to despawn I can get the same health on exit as a particle dboost, but the boost would be easier to time.

Reply:
160 heatframes + contact
185 heatframes + particle
250 heatframes  by waiting
And there's a neat double skree farm that is 210 heatframes + 2 drops

Reply:
Oh cool, I was able to reproduce that for the contact method; best I could get was 48 damage in the room total, so 38 heat damage = ~152 heat frames. 160 heat frames seem good to be doable consistently. With the particle boost I'm leaving with about 5 less energy. Seems good to put in the contact method, updated that now.